docs/package-server.md

```diff
@@ -10,7 +10,7 @@ The following are the parameters supported by this goal in addition to the [comm
 
 | Parameter | Description | Required |
 | --------  | ----------- | -------  |
-| packageFile | Location of the target file or directory. If the target location is a file, the contents of the server instance will be compressed into the specified file. If the target location is a directory, the contents of the server instance will be compressed into `${packageFile}/${project.build.finalName}.zip|jar` file. If the target location is not specified, it defaults to `${project.build.directory}/${project.build.finalName}.zip|jar`. A jar file is created when the packaging type is runnable. A zip file is created for other packaging types.| No |
+| packageFile | Location of the target file or directory. If the target location is a file, the contents of the server instance will be compressed into the specified file. If the target location is a directory, the contents of the server instance will be compressed into `${packageFile}/${project.build.finalName}.zip`&#124;`jar` file. If the target location is not specified, it defaults to `${project.build.directory}/${project.build.finalName}.zip`&#124;`jar`. A jar file is created when the packaging type is runnable. A zip file is created for other packaging types.| No |
```

Comment:
At first I wasn't sure why the extra ticks were needed on both sides of the | but it looks OK.

Also, I requested that the ticks be added to runnable near the end of that description but that was missed.  `runnable`